Formula/traildb.rb

```diff
@@ -0,0 +1,21 @@
+
+class Traildb < Formula
+  desc "Blazingly-fast database for log-structured data"
+  homepage "http://traildb.io"
+  url "https://github.com/traildb/traildb/archive/0.5.tar.gz"
+  sha256 "4d1b61cc7068ec3313fe6322fc366a996c9d357dd3edf667dd33f0ab2c103271"
+
+  depends_on "libarchive"
+  depends_on "homebrew/boneyard/judy"
```

Comment:
None of the known bugs trigger on OS X though. They might never trigger. Can we at least agree on a solution until those bugs trigger, if ever? 

For instance: what if I publish a judy fork (possibly patched for the Linux-only-for-now bug) in the traildb organization? Can I pull it and install it as a dependency of Traildb from within the Traildb formula? That's closer to having a vendored copy.